src/OmniSharp.DotNetTest/VSTestManager.cs

```diff
@@ -400,43 +401,57 @@ public override async Task<RunTestResponse> RunTestAsync(string[] methodNames, s
 
                         case MessageType.TestRunStatsChange:
                             var testRunChange = message.DeserializePayload<TestRunChangedEventArgs>();
-                            testResults.AddRange(testRunChange.NewTestResults);
+                            var newResults = ConvertResults(testRunChange.NewTestResults);
+                            passed = passed &&  testRunChange.NewTestResults.Any(o => o.Outcome == TestOutcome.Failed);
```

Comment:
I think passed should be true if previous runs were passing and none of the new results are failing.

```suggestion
                            passed = passed && !testRunChange.NewTestResults.Any(o => o.Outcome == TestOutcome.Failed);
```

Reply:
Huh, I've got this change in my fork, why is it not showing up here?
https://github.com/Samirat/omnisharp-roslyn/blob/master/src/OmniSharp.DotNetTest/VSTestManager.cs

Reply:
Oh this is from a branch, let me make sure this is all up to date.

Reply:
Nice, I had only fixed one of these